content/data/team_level_tips.csv

```diff
@@ -1,20 +1,20 @@
 teamLevel,requirement,action,totalStarters,totalAllStars,totalAllWorlds
-1 → 2,Rank 1 Level 2 x1,Level up 1 Starter to Rank 1 Level 2,Rank 1 Level 2 x1,,
-2 → 3,Rank 1 Level 2 x3,Level up 2 new Starters to Rank 1 Level 2,Rank 1 Level 2 x3,,
-3 → 4,Rank 1 Level 7 x3,Level up all 3 Starters from Rank 1 Level 2 to Rank 1 Level 7,Rank 1 Level 7 x3,,
-4 → 5,Rank 1 Level 10 x2,Level up 2 Starters from Rank 1 Level 7 to Rank 1 Level 10,"Rank 1 Level 10 x2, Rank 1 Level 7 x1",,
-5 → 6,Rank 2 x2,Rank up 2 Rank 1 Level 10 Starters to Rank 2,"Rank 2 x2, Rank 1 Level 7 x1",,
-6 → 7,Rank 2 x4,Rank up a new Starter and the remaining Rank 1 Level 7 Starter to Rank 2,Rank 2 x4,,
-7 → 8,Rank 1 Level 7 x10,Level up 6 new Starters to Rank 1 Level 7,"Rank 2 x4, Rank 1 Level 7 x6",,
-8 → 9,Rank 1 Level 7 x12,Level up 2 new Starters to Rank 1 Level 7,"Rank 2 x4, Rank 1 Level 7 x8",,
-9 → 10,Rank 1 Level 10 x9,Level up 5 Rank 1 Level 7 Starters to Rank 1 Level 10,"Rank 2 x4, Rank 1 Level 10 x5, Rank 1 Level 7 x3",,
-10 → 11,Rank 3 x2,Rank up 2 Rank 2 Starters to Rank 3,"Rank 3 x2, Rank 2 x2, Rank 1 Level 10 x5, Rank 1 Level 7 x3",,
-11 → 12,Rank 3 x4,Rank up 2 more Rank 2 Starters to Rank 3,"Rank 3 x4, Rank 1 Level 10 x5, Rank 1 Level 7 x3",,
-12 → 13,Rank 3 x7,Rank up 3 Rank 1 Level 10 Starters to Rank 3,"Rank 3 x7, Rank 1 Level 10 x2, Rank 1 Level 7 x3",,
-13 → 14,Rank 3 x10,Rank up 3 Rank 2 All-Star to Rank 3,"Rank 3 x7, Rank 1 Level 10 x2, Rank 1 Level 7 x3",Rank 3 x3,
-14 → 15,Rank 4 x2,Rank up 2 Rank 3 Starters to Rank 4,"Rank 4 x2, Rank 3 x5, Rank 1 Level 10 x2, Rank 1 Level 7 x3",Rank 3 x3,
-15 → 16,Rank 4 x4,Rank up 2 more Rank 3 Starters to Rank 4,"Rank 4 x4, Rank 3 x3, Rank 1 Level 10 x2, Rank 1 Level 7 x3",Rank 3 x3,
-16 → 17,Rank 4 x10,"Rank up 2 Rank 3 Starters, 3 Rank 3 All-Star and 1 All-World to Rank 4","Rank 4 x6, Rank 3 x1, Rank 1 Level 10 x2, Rank 1 Level 7 x3",Rank 4 x3,Rank 4 x1
-17 → 18,Rank 5 x2,Rank up 2 Rank 4 All-Star to Rank 5,"Rank 4 x6, Rank 3 x1, Rank 1 Level 10 x2, Rank 1 Level 7 x3","Rank 5 x2, Rank 4 x1",Rank 4 x1
-18 → 19,Rank 5 x4,Rank up the remaining Rank 4 All-Star and the Rank 4 All-World to Rank 5,"Rank 4 x6, Rank 3 x1, Rank 1 Level 10 x2, Rank 1 Level 7 x3",Rank 5 x3,Rank 5 x1
-19 → 20,Rank 6 x1,Rank up the Rank 5 All-World to Rank 6,"Rank 4 x6, Rank 3 x1, Rank 1 Level 10 x2, Rank 1 Level 7 x3",Rank 5 x3,Rank 6 x1
+1 → 2,Level 2 x1,Level up 1 Starter to Rank 1 Level 2 (R1L2),R1L2 x1,,
+2 → 3,Level 2 x3,Level up 2 new Starters to R1L2,R1L2 x3,,
+3 → 4,Level 7 x3,Level up all 3 Starters from R1L2 to R1L7,R1L7 x3,,
+4 → 5,Level 10 x2,Level up 2 Starters from R1L7 to R1L10,"R1L10 x2, R1L7 x1",,
+5 → 6,Rank 2 x2,Rank up 2 R1L10 Starters to Rank 2 (R2),"R2 x2, R1L7 x1",,
```

Comment:
`R2 x2` => `R2L1 x2`?

Reply:
yep thanks, too many to see them all haha